app/lib/util/files.js

```diff
@@ -110,35 +116,54 @@ function readFileAsJSON(file) {
 }
 
 /**
- * Find files by pattern in a list of search paths.
+ * Find files matching a given pattern in one or more search paths.
  *
- * @param {Object} options
- * @param {Array<string>} options.searchPaths
- * @param {string} options.pattern
+ * @param {string} pattern
+ * @param {Object} [options]
+ * @param {string[]} [options.searchPaths]
+ * @param {boolean} [options.absolute=true]
+ * @param {boolean} [options.nodir=true]
  *
- * @return {Array<string>} list of found, absolute path names
+ * @return {string[]}
  */
-function globFiles(options) {
+function globFiles(pattern, options = {}) {
+  const defaultOptions = {
+    absolute: true,
+    onlyFiles: true
+  };
 
   const {
     searchPaths,
-    pattern
+    ...otherOptions
   } = options;
 
-  return searchPaths.reduce((paths, searchPath) => {
-
-    const newPaths = globSync(pattern, {
-      cwd: searchPath,
-      nodir: true,
-      absolute: true
-    }).map(p => p.split(path.posix.sep).join(path.sep));
+  if (searchPaths) {
+    return searchPaths.reduce((paths, searchPath) => {
+      return [
+        ...paths,
+        ...globSync(toPosixPath(pattern), {
+          ...defaultOptions,
+          ...otherOptions,
+          cwd: toPosixPath(searchPath)
+        }).map(toPosixPath)
+      ];
+    }, []);
+  }
 
-    return [
-      ...paths,
-      ...newPaths
-    ];
-  }, []);
+  if (otherOptions.cwd) {
+    otherOptions.cwd = toPosixPath(otherOptions.cwd);
```

Comment:
Let's not leak side-effects here, but instead override `cwd` :arrow_down:.

Reply:
https://github.com/camunda/camunda-modeler/pull/3863/files/da9e89120b4a3dbe4e29ea6dfcd5cf076935db26#r1331353346 :o)